data-prepper-api/src/main/java/org/opensearch/dataprepper/logging/DataPrepperMarkers.java

```diff
@@ -0,0 +1,15 @@
+package org.opensearch.dataprepper.logging;
+
+import org.slf4j.Marker;
+import org.slf4j.MarkerFactory;
+
+public final class DataPrepperMarkers {
+    public static final Marker EVENT_MARKER = MarkerFactory.getMarker("EVENT");
```

Comment:
As the class is already `DataPrepperMarkers`, could we drop `_MARKER` from the names?